tests/cephadm/test_deploy_rgw_using_spec.py

```diff
@@ -0,0 +1,38 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+from cli.utilities.utils import create_yaml_config
+
+
+def run(ceph_cluster, **kw):
+    """
+    Validate the change in retention time for Prometheus
+    """
+    config = kw.get("config", {})
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+    specs = config.get("specs")
+    realm_name = specs.get("rgw_realm")
+
+    # Enable MGR module
+    CephAdm(installer).ceph.mgr.module.enable("rgw")
+
+    # Bootstrap rgw using new spec file
+    file = create_yaml_config(installer, specs)
+    conf = {"in-file": file}
```

Comment:
Hey @vinpapnoi 
One query, is this scenario similar to https://github.com/red-hat-storage/cephci/blob/master/tests/cephadm/test_verify_zone_group_rgw_with_cephadm.py

or it checks with different conf options? 
Please ignore if the scopes are different.

Reply:
@pranavprakash20 This test is specific to deploying RGW with a spec file. Above test uses the CLI args to deploy it using the `--realm-name`, `zonegroup-name`, `zone-name` parameters. Here we are using spec file with the `bootstrap` rgw realm option.

Reply:
Noted. Thanks for the clarification @vinpapnoi